packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -77,7 +77,7 @@ public function renderInputProps(Environment $environment, array $props, $allowe
         $transferringAttributes = [];
         foreach ($props as $propName => $propValue) {
             if (in_array($propName, self::VALIDATION_ATTRIBUTES, true) || in_array($propName, $allowedAttributes)) {
-                if (! is_null($propValue) && $propValue !== '') {
+                if (! is_null($propValue) && $propValue !== '' && $propValue !== false) {
```

Comment:
As @dlouhak already pointed out, I'm afraid it might be more complicated than this. Boolean and enum attributes are quite messy in HTML. Let's see how we go. 🤷🏻‍♂️

Reply:
I think there are multiple points of view but the question is "What is the correct way to disable an attribute?". Since there can be a use cases when the `false` value must be present, but I can be wrong.

So I am working with this:
- the value is true/exist -> `attribute="value"`
- the value is false -> `attribute="false"`
- the value is null/not-exist -> I can safely remove the attribute or do not display it

In most use cases is the `null` the same as the `false`. So do we know about some use case, when we need to apply an attribute with the `false` value? I can imagine only the data attributes.

Reply:
This was reverted.